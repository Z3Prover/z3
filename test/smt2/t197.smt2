
; Copyright (c) 2015 Microsoft Corporation
help)
(exit)