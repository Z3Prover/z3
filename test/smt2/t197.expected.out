(error "line 3 column 1: invalid command, '(' expected")
